apps/chat2/config_chat2.nim

```diff
@@ -237,11 +237,6 @@ type
       defaultValue: 0
       name: "rln-relay-cred-index" }: uint
 
-    rlnRelayMembershipGroupIndex* {.
-      desc: "the index of credentials to use, within a specific rln membership set",
-      defaultValue: 0
-      name: "rln-relay-membership-group-index" }: uint
-
```

Comment:
Before removing this directly, should we instead inform the users that this is going to be deprecated?

Reply:
note that rln feature is experimental, so imho for experimental features there is no need to deprecate first and then remove.

Reply:
I agree with this, also because we don't use this flag anywhere in the fleets either

Reply:
Ok thanks for the reminder. The same applies to `external_confi.nim` then